meta-luneos/recipes-connectivity/mobile-broadband-provider-info/mobile-broadband-provider-info/meson.build

```diff
@@ -24,6 +24,7 @@ pkgconfig.generate(
   dataonly: true,
 )
 
+if false
```

Comment:
@tofee Not sure why you disabled this bit?

Reply:
After a check, the apns-conf.xsl input file doesn't exist yet in the sailfishos repo, so that command doesn't work here.
After a second thought, I wonder if we shouldn't add the missing input file, rather than disabling this output...